federation/src/client/1_0/AuthenticationClient.ts

```diff
@@ -54,10 +54,10 @@ export class AuthenticationClient {
       const { data } = await this.client.rawRequest<any>(authenticate, { args })
       methodLogger.debug('after authenticate: data:', data)
 
-      const authUuid: string = data?.authenticate
-      if (authUuid) {
-        methodLogger.debug('received authenticated uuid', authUuid)
-        return authUuid
+      const responseJwt: string = data?.authenticate
```

Comment:
```suggestion
      const responseJwt = data?.authenticate
```
string type isn't needed here, if type was correct in this.client.rawRequest defined.